src/windows/ride_list.c

```diff
@@ -459,7 +459,7 @@ static void window_ride_list_invalidate(rct_window *w)
 	w->widgets[WIDX_CLOSE].left = w->width - 13;
 	w->widgets[WIDX_CLOSE].right = w->width - 3;
 	w->widgets[WIDX_LIST].right = w->width - 26;
-	w->widgets[WIDX_LIST].bottom = w->height - 4;
+	w->widgets[WIDX_LIST].bottom = w->height - 0x0F;
```

Comment:
I agree, the reason I did that was because it was the same in [staff_list.c#L516](https://github.com/OpenRCT2/OpenRCT2/blob/b5706a5de896b27cbd67f671c8c202532595f5e1/src/windows/staff_list.c#L516), so I wanted to be consistent. I'll make a commit to change them both.